spec/imsc-hrm.html

```diff
@@ -245,7 +245,8 @@ <h2>Terms and Definitions</h2>
     </p>
     <p>
       A sequence of consecutive <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Documents</dfn>
```

Comment:
Do we need to say that the ISDs shall be generated from one or more syntactically conformant IMSC documents?

Reply:
See if this is clearer.